Dockerfile

```diff
@@ -74,12 +74,15 @@ COPY db ${APP_HOME}/db
 COPY app ${APP_HOME}/app
 COPY spec ${APP_HOME}/spec
 
+# Not actual SECRET_KEY_BASE and is randomly generated just for this task
+
 RUN DFE_SIGN_IN_API_CLIENT_ID= \
   DFE_SIGN_IN_API_SECRET= \
   DFE_SIGN_IN_API_ENDPOINT= \
   ADMIN_ALLOWED_IPS= \
   ENVIRONMENT_NAME= \
   SUPPRESS_DFE_ANALYTICS_INIT= \
+  SECRET_KEY_BASE="1d40d61e24db3cd4282716c45890c33b5be4ac99eee12ddfe758a01ad03d3206b5888911211ea9e29eee891c76e7011e8eae9f0c868d500575ea3b18749f31f1" \
```

Comment:
I think we can also do `SECRET_KEY_BASE_DUMMY=1` but this is fine tbh

Reply:
never heard of this before and first time I've seen this. will give it a go

Reply:
done